sources/dicom/large_image_source_dicom/assetstore/dicomweb_assetstore_adapter.py

```diff
@@ -190,6 +190,10 @@ def importData(self, parent, parentType, params, progress, user, **kwargs):
             item = Item().createItem(name=series_uid, creator=user, folder=folder,
                                      reuseExisting=True)
 
+            # Set the DICOMweb metadata
+            item['dicomweb_meta'] = get_dicomweb_metadata(client, study_uid, series_uid)
+            Item().save(item)
```

Comment:
We should probably use `item = Item().save(item)`.  The item _could_ be altered via some girder event, so it is considered good form to do this to ensure that the returned item reflects this if it happened.

Reply:
Fixed!